Geometry/HGCalMapping/test/HGCalMappingESSourceTester.cc

```diff
@@ -49,7 +51,8 @@ HGCalMappingESSourceTester::HGCalMappingESSourceTester(const edm::ParameterSet&
       cellTkn_(esConsumes()),
       moduleIndexTkn_(esConsumes()),
       moduleTkn_(esConsumes()),
-      denseIndexTkn_(esConsumes()) {}
+      denseIndexTkn_(esConsumes()),
+      verbosity_(iConfig.getParameter<int>("verbosity")) {}
```

Comment:
should be an untracked parameter since it does not affect core functionality or calculations

Reply:
OK